docs/roadmap/index.md

```diff
@@ -1,5 +1,8 @@
 # Roadmap
 
+!!! heart "Timetable"
+    Please see the [Migration and Suport Timetable](../migration/timetable.md) for our commitment to end users.
```

Comment:
Please move this phase to the below box: 

> The Flux custom resource definitions which are at `v1beta1` and `v2beta1` and their controllers are considered stable and production ready. Please see the [Migration and Suport Timetable](../migration/timetable.md) for more details.

Reply:
Does this look good now?